public/app/features/commandPalette/KBarResults.tsx

```diff
@@ -139,21 +142,16 @@ export const KBarResults = (props: KBarResultsProps) => {
 
   const pointerMoved = usePointerMovedSinceMount();
 
+  const styles = useStyles2(getStyles);
+
   return (
-    <div
-      ref={parentRef}
-      style={{
-        maxHeight: props.maxHeight || 400,
-        position: 'relative',
-        overflow: 'auto',
-      }}
-    >
+    <div ref={parentRef} className={styles.wrapper}>
       <div
         role="listbox"
         id={KBAR_LISTBOX}
+        className={styles.listBox}
         style={{
           height: `${rowVirtualizer.totalSize}px`,
```

Comment:
Currently there two different ways to apply styling. Since you already created the listBox object could you please add this as well?